### PR TITLE
Japanese support

### PR DESCRIPTION
**Pull request by malken21**
Added Japanese language support.

image

### CODE REVIEW

**Review by elvissteinjr — CHANGES_REQUESTED** [review]
Thank you for your efforts!
While the NewUI branch is still going to get some additions with new translation strings, I don't see a reason to reject this. 
I'll also add some extra mention for community provided translations with authorship in the application itself when I have time.

If you're interested, the "input" folder also has translatable strings for the SteamVR controller binding screen (new files need to be added in "action_manifest.json" then as well).

I only really have the one nitpick with the name field. This seems good to me otherwise (as far as I am able to tell anyways).

File: assets/lang/ja.ini [hunk 1]
```diff
@@ -0,0 +1,596 @@
+[TranslationInfo]
+Name=Japanize
```
[the remark this answers is not in the document]
Comment (malken21) [reply assets/lang/ja.ini:2]: I made a spelling mistake. Thank you for checking.

**Review by elvissteinjr — COMMENTED** [review]
I apologize for the second review here.
While I still can't check the Japanese itself as I don't speak it, I got curious and ran some machine translation back to English for a few sections. Of course this can't be fully trusted and has inaccuracies, but I still noticed two more things there.

I also noticed that I still had unpushed commits on my machine that added translation strings. Sorry about that.
This will still be unavoidable going forward and I'm not asking you to keep up with every change. Just wanted to let you know.

File: assets/lang/ja.ini [hunk 2]
```diff
@@ -0,0 +1,596 @@
...
+tstr_SettingsEnvironmentBackgroundColor=背景の色
+tstr_SettingsEnvironmentBackgroundColorDispModeNever=常に非表示
+tstr_SettingsEnvironmentBackgroundColorDispModeDPlusTab=Desktop+ タブでのみ表示
+tstr_SettingsEnvironmentBackgroundColorDispModeAlways=常に表示
+tstr_SettingsEnvironmentDimInterface=インターフェイスを暗くする
+tstr_SettingsEnvironmentDimInterfaceTip=Desktop+ のダッシュボードタブを開いている間、SteamVR のダッシュボードと Desktop+ のインターフェイスを暗くします。
+
+tstr_SettingsProfilesOverlays=オーバーレイのプロファイル
+tstr_SettingsProfilesApps=アプリケーションプロファイル
+tstr_SettingsProfilesManage=プロファイル管理
```
[the remark this answers is not in the document]
Comment (malken21) [reply assets/lang/ja.ini]: I decided not to mention profiles.

File: assets/lang/ja.ini [hunk 3]
```diff
@@ -0,0 +1,596 @@
...
+
+tstr_SettingsTroubleshootingRestart=再起動
+tstr_SettingsTroubleshootingRestartSteam=Steam再起動
+tstr_SettingsTroubleshootingRestartDesktop=デスクトップモードで再起動
+tstr_SettingsTroubleshootingElevatedModeEnter=管理者モード 有効
+tstr_SettingsTroubleshootingElevatedModeLeave=管理者モード 無効
+tstr_SettingsTroubleshootingSettingsReset=初期設定に戻す
+tstr_SettingsTroubleshootingSettingsResetConfirmDescription=デフォルト設定を復元すると、選択した要素がデフォルト値にリセットされます。\nこれは、元に戻すことはできません。\n\nリセットする値:
+tstr_SettingsTroubleshootingSettingsResetConfirmButton=選択している項目をリセット
+tstr_SettingsTroubleshootingSettingsResetConfirmElementOverlays=Current オーバーレイセットアップ
```
[the remark this answers is not in the document]
Comment (malken21) [reply assets/lang/ja.ini]: I apologize. I neglected to translate that part.

**Comment by elvissteinjr** [comment]
Looks good to me now. Thanks again for your efforts!

**Comment by elvissteinjr** [comment]
Hi, I've recently pushed a few commits that added new translation strings. 
If you have the time, it would be great if these could be added to your Japanese translation as well.

I'm not looking to bother you like this every time new strings come up (this will inevitably keep happening), but I thought it would be good to at least have the translation be complete for the release it debuts in.
The next NewUI Preview build is planned to be released up soon. There are still some things I want to work on before this happens (so no need to rush), but nothing that would add more strings.

There's also a new "Author" field in the TranslationInfo section of translation files. This is displayed when the language is chosen and used to mark the translation as a community one. 
Feel free to put whatever name you're comfortable to be credited as in there.

I've also attached a version of the Japanese language file with the new strings already inserted in English, in case this helps (as .txt since GitHub doesn't allow .ini upload). 
If you prefer not to use it, keep in mind I've adjusted the naming scheme and removed some strings that weren't used anywhere, so it's not the exact same file from last time.
This would probably need to happen in a separate pull request I think? Not really sure to be honest.

Thanks!
[ja_new.txt](https://github.com/elvissteinjr/DesktopPlus/files/14188378/ja_new.txt)

**Comment by elvissteinjr** [comment]
I'm fairly close to wrapping up for real this time, so I'll try to ask again.
I have attempted to contact you via the Twitter account on your profile, but I'm not sure how active you are on there and that site has failed to notify me of DMs before.

Suppose I'm looking somewhat desperate here for volunteer work, but I'd just find it cool to be able to announce full Japanese support for the 3.0 release.
Well, if you can't that's fine of course, don't get me wrong. Just want to at least have tried properly before looking for alternatives.

Thanks again.

**Pull request merged**